sdks/python/apache_beam/coders/typecoders.py

```diff
@@ -145,7 +146,9 @@ def verify_deterministic(self, key_coder, op_name, silent=True):
                    'and for custom key classes, by writing a '
                    'deterministic custom Coder. Please see the '
                    'documentation for more details.' % (key_coder, op_name))
-      if isinstance(key_coder, (coders.PickleCoder, self._fallback_coder)):
+      # TODO(vikasrk): Should we include other fallback coders?
+      if isinstance(key_coder, (coders.PickleCoder,
+                                coders.FastPrimitivesCoder)):
```

Comment:
Robert: do we really want to replace FastPrimitivesCoder with DeterministicPickleCoder?  I think the only thing here should be PickleCoder, and including FastPrimitivesCoder is an oversight.